src/pages/guides/understanding-pdas.md

```diff
@@ -0,0 +1,104 @@
+---
+title: Understanding Solana Program Derived Addresses (PDAs)
+metaTitle: Understanding Solana Program Derived Addresses (PDAs)
+description: Learn about Solana Program Derived Addresses (PDAs) and their use cases.
+# remember to update dates also in /components/guides/index.js
+created: '06-16-2024'
+updated: '06-21-2024'
+---
+
+## Overview
+**Program Derived Addresses (PDAs)** are special types of account used on Solana that are deterministically derived and look like standard public keys, but have no associated private keys.
+
+## Key Characteristics of PDAs
+- **Deterministic**: PDAs are deterministically derived from a combination of a program ID and "seeds" (predefined inputs) values
+- **Secure**: no external user can generate a valid signature for the PDA address. Only the program that derived the PDA can sign transactions involving it.
+- **Convenient**: They simplify the management of program-specific state and data by providing a consistent way to address accounts.
+
+## Role of PDAs
+PDAs are primarily used to:
+
+- **Manage State**: PDAs allow programs to store data in a determinstic address, making reading that state back easy.
+- **Authorize Transactions**: Only the program that owns the PDA can authorize transactions involving it, ensuring controlled access.
```

Comment:
Can you explain why PDAs cannot be controlled by a private key here?

The signing transactions parts also relates to my other comment on CPI.